js-classes/inheritance-es5.js

```diff
@@ -0,0 +1,79 @@
+let date = new Date();
+function Car (name, model, year, color, maxSpeed, fuelCapacity = 60, fuelConsumption = 10){
```

Comment:
In the old JS it was not possible to set default values for parameters instead you could do like this
```
 this.fuelCapacity = fuelCapacity || 60
```